core/config.py

```diff
@@ -103,6 +103,9 @@ def __setitem__(self, key, value):
     # Unless overridden by the Project: Use english as default language
     "viur.defaultLanguage": "en",
 
+    # If disabled the local logging will not send with requestLogger to the cloud
+    "viur.dev_server_cloud_logging": True,
```

Comment:
```suggestion
    "viur.dev_server_cloud_logging": False,
```
shouldn't this be False by default?